### PR TITLE
Improve code coverage for cuts.py

### PR DESCRIPTION
**Pull request by cpurmessur**
Here I added some tests to test_cuts.py to improve code coverage. It should be at 100% now. I also change the term "sequence" to"container" in cuts.py. This is because the documentation refers to the inputs of the functions as sets and sets are unordered and therefore are not sequences.

### CODE REVIEW

File: networkx/algorithms/cuts.py [hunk 1]
```diff
@@ -32,11 +32,11 @@ def cut_size(G, S, T=None, weight=None):
     ----------
     G : NetworkX graph
 
-    S : sequence
-        A sequence of nodes in `G`.
+    S : container
```
Comment (rossbar) [inline networkx/algorithms/cuts.py:35]: I'm probably being overly pedantic, but I don't think `container` is the right description for the parameter. In this case, `S` and `T` are passed to `nx.edge_boundary`, which expects Iterables for the corresponding nbunch inputs. In the [Python ABC hierarchy](https://docs.python.org/3/library/collections.abc.html#module-collections.abc), Iterables are derived from containers and this function wouldn't work if you passed in an object that *only* implemented `__contains__`. 

I *think* "collection" is probably the more accurate term, but please double check against the linked abc docs and the other instances to make sure this makes sense.

File: networkx/algorithms/tests/test_cuts.py [hunk 2]
```diff
@@ -104,6 +115,13 @@ def test_graph(self):
         conductance = nx.conductance(G, S, T)
         expected = 1 / 5
         assert expected == conductance
+        # Test with no input T
+        G2 = nx.barbell_graph(3, 0)
+        # There is only one cut edge, and each set has volume seven.
+        S2 = {0, 1, 2}
+        conductance_no_T = nx.conductance(G2, S2)
+        expected2 = 1 / 7
+        assert expected2 == conductance_no_T
```
Comment (rossbar) [inline networkx/algorithms/tests/test_cuts.py:124]:
```suggestion
        assert nx.conductance(G2, S2) == 1 / 7
```
A slightly more condense way of putting this - feel free to ignore!

**Review by rossbar — COMMENTED** [review]
+1 on the changes to the tests (feel free to ignore my nits there).

Re: the docstring changes - I think "container" should be "collection" (see my comment on the first instance)

File: networkx/algorithms/cuts.py (hunk 1 above)
Comment (cpurmessur) [reply networkx/algorithms/cuts.py]: "Collection" makes sense-- I updated the docstring with the new commit. Thanks!

File: networkx/algorithms/tests/test_cuts.py (hunk 2 above)
Comment (cpurmessur) [reply networkx/algorithms/tests/test_cuts.py]: I also made tests more concise for this and a couple of other test cases

**Review by rossbar — COMMENTED** [review]
Thanks for the touch-ups, this LGTM. I think collection is an appropriate term for the parameter descriptions, but I'm not an `abc` expert - at the very least I do think it's more accurate than `sequence`. Thanks @cpurmessur !

**Review by rossbar — APPROVED** [review]

**Review by dschult — APPROVED** [review]

**Pull request merged**